src/js/page.js

```diff
@@ -336,14 +338,44 @@ export default {
     },
     _getTocHtmlFragment( tocItems ) {
         const items = document.createDocumentFragment();
+        let currentGroupList;
         tocItems.forEach( item => {
+            const $itemPageEl = $( item.pageEl );
             const li = document.createElement( 'li' );
-            const a = document.createElement( 'a' );
-            a.setAttribute( 'href', `#${item.pageEl.querySelector( '[name]' ).getAttribute( 'name' )}` );
-            a.textContent = item.tocItemText;
-            li.append( a );
-            items.appendChild( li );
+            if ( $itemPageEl.is( '.or-group' ) ) {
+                const details = document.createElement( 'details' );
+                const summary = document.createElement( 'summary' );
+                const groupList = document.createElement( 'ul' );
+                summary.textContent = item.tocItemText;
+                $( details ).addClass( 'groups-toc' );
```

Comment:
we're slowly removing jQuery, so let's use `details.classList.add('groups-toc')`.